includes/reports/overview/list-table/class-sensei-reports-overview-list-table-courses.php

```diff
@@ -32,10 +32,18 @@ class Sensei_Reports_Overview_List_Table_Courses extends Sensei_Reports_Overview
 	/**
 	 * Sensei course related services.
 	 *
+	 * @var array
+	 */
+	public $total_courses_ids = [];
```

Comment:
Is this variable needed?

Reply:
Nope: https://github.com/Automattic/sensei/pull/5091/commits/196d209eecf119a94a26e15eb7fb0d7d48a8e307